### PR TITLE
Ask users to open Android issues in respective forum

### PR DESCRIPTION
**Pull request by bmuschko**

### CODE REVIEW

**Comment by big-guy** [comment]
LGTM @bmuschko

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <!--- Provide a brief summary of the issue in the title above -->
+<!--- We'd kindly ask you to open any issues related to the Gradle Android plugin or Android studio on the Google forum at `https://source.android.com/source/report-bugs` -->
```
Comment (jjohannes) [inline .github/ISSUE_TEMPLATE.md]: ❌ Wording:

Android studio -> Android Studio
Google forum ->  Android Issue Tracker

⭕️ I think we can also add (assuming that the more precise the instructions are the more likely people follow them):
...in the components _App Development > Gradle_ or _App Development > Android Studio > Gradle IDE_.
(see component selection here https://issuetracker.google.com/issues/new)

Comment (bmuschko) [inline .github/ISSUE_TEMPLATE.md]: Can you elaborate on your second suggestion? I think users would still have select between `Android Public Tracker > App Development > Android Studio > Gradle IDE` and `Android Public Tracker > App Development > Gradle`. If those options change then our suggestion would get out of date. I am voting for not going into details. The page we reference explains all details.

Comment (bmuschko) [inline .github/ISSUE_TEMPLATE.md]: Fixed your first comment.

**Comment by bmuschko** [comment]
@jjohannes I went ahead with the merge. Feel free to directly edit the file if you would like to incorporate additional changes.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (jjohannes) [inline .github/ISSUE_TEMPLATE.md]: I just thought maybe it stresses the point that there is a specific section for **Android+Gradle** section in that Issue Tracker, and that is not only about general Android stuff. But it's probably, as you say, too much detail.

I am fine with not adding that.

**Pull request merged**